src/test/java/com/arangodb/ArangoDatabaseTest.java

```diff
@@ -973,6 +1023,23 @@ public void createGraph() {
         assertThat(result.getName(), is(name));
     }
 
+    @Test
+    public void createGraphSatellite() {
+        assumeTrue(isAtLeastVersion(3, 7));
+        assumeTrue(isCluster());
+        assumeTrue(isEnterprise());
+
+        String name = "graph-" + rnd();
+        final GraphEntity result = db.createGraph(name, null, new GraphCreateOptions().satellite(true));
+        assertThat(result.getSatellite(), is(true));
+
+        GraphEntity info = db.graph(name).getInfo();
+        assertThat(info.getSatellite(), is(true));
+
+        GraphEntity graph = db.getGraphs().stream().filter(g -> name.equals(g.getName())).findFirst().get();
+        assertThat(graph.getSatellite(), is(true));
+    }
```

Comment:
Good, but why not implemented in `src/test/java/com/arangodb/ArangoGraphTest.java` file?

Reply:
Thanks for spotting, graph creation tests seem to be randomly placed in both files... 
I will refactor this soon in a separate PR.